### PR TITLE
Peer programming with Andrew and eating goldfish

### PR DESCRIPTION
**Pull request by christylaguardia**
#### Single-line Summary
**Today, Christy and Andrew paired together. It took about 1 1/2 hours**

#### Reflect and summarize on your process for each `TODO` item :  
  1. First, we started on the script tag for the handlebars
  2. Next, we add the script tag for the handlebars template
  3. Then, we ate goldfish
  4. Then, we used handlebars in article.js to get the template and compile
  5. Then, we used the function from handlebars to fill in the new template article
  6. Then, we added the data-author attribute to the template, so that the filter would work

#### Checklist (before submitting, fill in each set of square brackets with an 'x')
- [x] We have titled the Pull Request similar to our branch name (ex: 'brian-rick'). 
- [x] This PR includes commits from both myself and my partner; e.g. We followed good pair programming practices by switching driver/navigator roles.
- [x] There is no extraneous, unrelated code included in this PR.
- [x] We have summarized our `TODO:` process above.

### CODE REVIEW

File: starter-code/index.html [hunk 1]
```diff
@@ -19,8 +19,25 @@
         - If your object doesn't have values you need yet, you can add those properties in the model.
     -->
 
+    <script id="template" type="text/x-handlebars-template">
+      <section>
```
Comment (kbrumer) [inline starter-code/index.html]: You do not want a `section` tag - just the `article`.

File: starter-code/scripts/article.js [hunk 2]
```diff
@@ -14,6 +14,8 @@ function Article (opts) {
 Article.prototype.toHtml = function() {
   // TODO: Use handlebars to render your articles.
   //       - Get your template from the DOM and "compile" your template with Handlebars.
+  var myCompileFunction = Handlebars.compile($('#template').html());
```
Comment (kbrumer) [inline starter-code/scripts/article.js]: `template` or `templateFunction` might be a better name.

**Comment by kbrumer** [comment]
👍

File: starter-code/scripts/article.js [hunk 3]
```diff
@@ -23,7 +25,7 @@ Article.prototype.toHtml = function() {
   this.publishStatus = this.publishedOn ? `published ${this.daysAgo} days ago` : '(draft)';
 
   // TODO: Use the function that Handlebars gave you to return your filled-in html template for THIS article.
-
+  $('#articles').append(myCompileFunction(this));
```
Comment (JPGothro) [inline starter-code/scripts/article.js]: You don't want to append here, just return the result of the handlebars compile.

**Pull request closed without merging**